### PR TITLE
Persons mentioned in book tab

### PR DESCRIPTION
**Pull request by miri**
#67

### CODE REVIEW

**Comment by miri** [comment]
image

**Comment by miri** [comment]
image

File: haskala/modules/custom/haskala_book/haskala_book.module [hunk 1]
```diff
@@ -95,3 +95,51 @@ function haskala_book_get_tab_fields($wrapper, $tab, $node_view, $book_nid) {
...
+  $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'mention')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->fieldCondition('field_book', 'target_id', $book_nid);
+  $result = $query->execute();
+  if (empty($result['node'])) {
+    return;
+  }
+  $mention_nodes = node_load_multiple(array_keys($result['node']));
+  if ($mention_nodes) {
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/haskala_book.module]:
```if (!$mention_nodes) {
return;
}```

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 2]
```diff
@@ -57,6 +57,14 @@ function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $c
   // Add Book producers tab.
   haskala_book_production_tab_fields($tabs_fields, $book_nid);
 
+  // Add mentionees for PERSONS MENTIONED IN BOOK tab.
+    $mentionees = haskala_book_get_mentions($book_nid);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Fix indentation

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 3]
```diff
@@ -57,6 +57,14 @@ function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $c
...
+    if ($mentionees) {
+      $tabs_fields['Persons mentioned in book']['sub_title'] = 'People mentioned in the book';
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Use t()

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 4]
```diff
@@ -57,6 +57,14 @@ function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $c
...
+      $mention_tmp = array_merge($tabs_fields['Persons mentioned in book'], $mentionees);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: t() also here

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php [hunk 5]
```diff
@@ -43,12 +43,19 @@
...
         <div class="fields">
-          <?php foreach($multi_tab_fields as $single_tab_fields): ?>
-            <div class="fields-multi">
-              <?php foreach($single_tab_fields as $tab_field): ?>
-                <?php print render($tab_field); ?>
-              <?php endforeach; ?>
-            </div>
+          <?php foreach($multi_tab_fields as $key => $single_tab_fields):
+            // Sub title in tab if exist.?>
+            <?php if ($key === 'sub_title'): ?>
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php]: Change to ==

**Comment by miri** [comment]
Merged.

**Pull request closed without merging**